src/big-query-functions/big-query-service-class.ts

```diff
@@ -0,0 +1,58 @@
+import { Issue } from "../api-functions/get-issues-by-project";
+import { BigQuery } from "@google-cloud/bigquery";
+
+export default class BQ {
+    //Creates a new table in the dataset
```

Comment:
file name should be big-query.service.ts

Directory name should be only "big-query"